tools/predict_net.py

```diff
@@ -0,0 +1,207 @@
+from time import time
+
+import numpy as np
+import pandas as pd
+import cv2
+import torch
+
+import slowfast.utils.checkpoint as cu
+import slowfast.utils.distributed as du
+import slowfast.utils.logging as logging
+import slowfast.utils.misc as misc
+from slowfast.datasets import loader
+from slowfast.models import model_builder
+# from slowfast.utils.meters import TestMeter
+
+logger = logging.get_logger(__name__)
+
+def imresize(im, dsize):
+    '''
+    Resize the image to the specified square sizes and 
+    maintain the original aspect ratio using padding.
+
+    Args:
+
+        im -- input image.
+        dsize -- output sizes, can be an integer or a tuple.
+
+    Returns:
+
+        resized image.
+    '''
+    if type(dsize) is int:
+        dsize = (dsize, dsize)
+        
+    im_h, im_w, _ = im.shape
+    to_w, to_h = dsize
+    scale_ratio = min(to_w/im_w, to_h/im_h)
+    new_im = cv2.resize(im,(0, 0), 
+                        fx=scale_ratio, fy=scale_ratio, 
+                        interpolation=cv2.INTER_AREA)
+    new_h, new_w, _ = new_im.shape
+    padded_im = np.full((to_h, to_w, 3), 128)
+    x1 = (to_w-new_w)//2
+    x2 = x1 + new_w
+    y1 = (to_h-new_h)//2
+    y2 = y1 + new_h
+    padded_im[y1:y2, x1:x2, :] = new_im 
+    
+    return padded_im
```

Comment:
Seems to me that we could follow what we have in `datasets/kinetics.py`?
Something like
```
            frames = transform.random_short_side_scale_jitter(
                frames, dsize, dsize
            )
```
I am not sure how would it perform with mean padding: normally the input is without padding, which means the model never seen a padded image. 

I feel two options might lead a better choice (performance?):

1) center crop:
```
            frames = transform.uniform_crop(frames, crop_size, spatial_idx)
```

or 
2) fully convolutional testing - ideally this would give us a better performance by feeding the frames without padding or cropping, then having a global pooling in the end of the network, which is implemented in [here](https://github.com/facebookresearch/SlowFast/blob/cddf2ea7c1b0fc4452e8269b977fc11a09ad2387/slowfast/models/head_helper.py#L90-L91):

I could also refactor this part later if you prefer to keep it.

Reply:
2) is a good idea, I'm thinking of using slowfast.datasets.cv2_transform.scale. What do you think?

Reply:
2 ) is a good idea, I'm thinking of using slowfast.datasets.cv2_transform.scale. What do you think?